### PR TITLE
Adding files to Java-playwright example project

### PR DESCRIPTION
**Pull request by ankit-dq**

### CODE REVIEW

File: Java/playwright/playwright.iml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Zidious) [inline Java/playwright/playwright.iml:1]: This can be removed or adding to `.gitignore`

File: Java/playwright/pom.xml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  </repositories>
+
+  <name>playwright</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```
Comment (Zidious) [inline Java/playwright/pom.xml:24]: Playwright requires minimum of Java8

File: Java/playwright/src/test/java/org/example/example_Test.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        if (file.exists()){
+            FileUtils.deleteDirectory(file);
+        }
+    }
+
+    @Test
+    public void test1(){
+        page.navigate("https://broken-workshop.dequelabs.com/");
+        AxeResults axeResults = axePlaywrightBuilder.analyze();
+    }
```
Comment (Zidious) [inline Java/playwright/src/test/java/org/example/example_Test.java:41]: We are not asserting anything in this test, also the test name does not reflect what the test is doing: 
```suggestion
    @Test
    public void isBrokenWorshopAccessible(){
        page.navigate("https://broken-workshop.dequelabs.com/");
        AxeResults axeResults = axePlaywrightBuilder.analyze();
        
        assertTrue(axeResults.violationFree());
    }
```

File: Java/playwright/src/test/java/org/example/example_Test.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+    @Test
+    public void test2(){
+        page.navigate("https://broken-workshop.dequelabs.com/");
+        page.locator("#main-content > div.Recipes > div:nth-child(1) > div.Recipes__card-foot > button").click();
+        AxeResults axeResults2 = axePlaywrightBuilder.analyze();
+    }
```
Comment (Zidious) [inline Java/playwright/src/test/java/org/example/example_Test.java:48]:
```suggestion
    @Test
    public void isRecipeCardAccessible(){
        page.navigate("https://broken-workshop.dequelabs.com/");
        page.locator("#main-content > div.Recipes > div:nth-child(1) > div.Recipes__card-foot > button").click();
        AxeResults axeResults = axePlaywrightBuilder.analyze();
        
        assertTrue(axeResults.violationFree());
    }
```

**Review by Zidious — CHANGES_REQUESTED** [review]
Minor changes

**Review by Zidious — APPROVED** [review]
LGTM - @jthickens will need to review the README

File: Java/playwright/pom.xml [hunk 5]
```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.example</groupId>
+  <artifactId>playwright</artifactId>
+  <version>1.0-SNAPSHOT</version>
```
Comment (jthickens) [inline Java/playwright/pom.xml:9]: is this specifying the version of our playwright java integration? Shouldn't this be 4.4?

**Review by jthickens — CHANGES_REQUESTED** [review]

File: Java/playwright/pom.xml (hunk 5 above)
Comment (ankit-dq) [reply Java/playwright/pom.xml:9]: Updated the version. Its should be 4.2.2, I guess that's the one being used in prod currently

Comment (Zidious) [reply Java/playwright/pom.xml]: Sorry missed this. versions denoted as `x-SNAPSHOT` are QA builds. We should use the latest release on Agora should be `4.4.0`

File: Java/playwright/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+cd Java/playwright
+```
+
+## Binary Reporter
+
+Download the respective [Binary reporter](https://docs.deque.com/devtools-html/4.0.0/en/downloads#binary-reporter) for your operating system and place it in the **_src/test/resources_** directory.
+
+## Install Dependencies
+
+Install the dependencies including **axe DevTools Java Selenium** and **Playwright** for the project.
```
Comment (michael-siek) [inline Java/playwright/README.md:28]:
```suggestion
Install the dependencies including **axe DevTools Java Playwright** and **Playwright** for the project.
```

**Review by michael-siek — CHANGES_REQUESTED** [review]

**Review by michael-siek — APPROVED** [review]

**Pull request merged**